config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -172,6 +172,18 @@ spec:
                 by components like the web console to tell users where to find the
                 Kubernetes API.
               type: string
+            controlPlaneTopology:
+              description: ControlPlaneTopology express the expectations for operands
+                that normally run on control nodes. The default is 'HighlyAvailable',
+                which represents the behavior operators have in a \"normal\" cluster.
+                The 'SingleReplica' mode will be used in single-node deployments (developer
+                and production) for example, and the operators should not configure
+                the operand for highly-available operation
+              type: string
+              enum:
+              - ""
```

Comment:
why do we need empty? Can't we default to HighlyAvailable?

Reply:
https://github.com/openshift/api/pull/827#discussion_r546936041

Reply:
that comment does not explain why we don't just use a default, making the API easier to use and observe.

Reply:
@staebler can you elaborate on why do you think we should have the empty value and not a default?

Reply:
The empty option was removed